app/experimenter/experiments/constants/nimbus.py

```diff
@@ -206,6 +206,21 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_FIRST_RUN_WINDOWS_10_NEWER_FX100_NEWER = NimbusTargetingConfig(
+    name="Fx100+ first start-up users on Windows 10 Creators (build 15063) or newer",
+    slug="first_run_win10_fx100",
+    description="Fx100+ first start-up users (e.g. for about:welcome) on Windows 1507+",
+    targeting=("{first_run} "
+               "&& os.windowsBuildNumber >= 15063 "
+               "&& (version|versionCompare('100.!') >= 0)"
```

Comment:
No need to hardcode the version as the advanced targeting can be combined with "regular" audience targeting, e.g., min version, channel, locales.